### PR TITLE
chrono: Pre soft launch

### PR DESCRIPTION
**Pull request by anastasia**
- read only / author mode responsive
- date autofill fix
- history mode and base url
- search fix
- added court, removed reporter
- moved all dropdowns to v-select module
- bumped docker

### CODE REVIEW

**Comment by bensteinberg** [comment]
This is failing on the deploy step, at `docker-compose exec web yarn build`:

> These dependencies were not found:
> 
> * vue-select in /node_modules/cache-loader/dist/cjs.js??ref--12-0!/node_modules/thread-loader/dist/cjs.js!/node_modules/babel-loader/lib!/node_modules/cache-loader/dist/cjs.js??ref--0-0!/node_modules/vue-loader/lib??vue-loader-options!./static/js/labs/chronolawgic/add-case-modal.vue?vue&type=script&lang=js&, /node_modules/cache-loader/dist/cjs.js??ref--12-0!/node_modules/thread-loader/dist/cjs.js!/node_modules/babel-loader/lib!/node_modules/cache-loader/dist/cjs.js??ref--0-0!/node_modules/vue-loader/lib??vue-loader-options!./static/js/labs/chronolawgic/add-event-modal.vue?vue&type=script&lang=js&
> * vue-select/dist/vue-select.css in /node_modules/cache-loader/dist/cjs.js??ref--12-0!/node_modules/thread-loader/dist/cjs.js!/node_modules/babel-loader/lib!/node_modules/cache-loader/dist/cjs.js??ref--0-0!/node_modules/vue-loader/lib??vue-loader-options!./static/js/labs/chronolawgic/add-event-modal.vue?vue&type=script&lang=js&
> 
> To install them, you can run: npm install --save vue-select vue-select/dist/vue-select.css

File: capstone/yarn.lock [hunk 1]
```diff
@@ -9245,6 +9245,11 @@ vue-router@^3.0.2:
   resolved "https://registry.yarnpkg.com/vue-router/-/vue-router-3.4.9.tgz#c016f42030ae2932f14e4748b39a1d9a0e250e66"
   integrity sha512-CGAKWN44RqXW06oC+u4mPgHLQQi2t6vLD/JbGRDAXm0YpMv0bgpKuU5bBd7AvMgfTz9kXVRIWKHqRwGEb8xFkA==
 
+vue-select@^3.11.2:
```
Comment (ChefAndy) [inline capstone/yarn.lock:9248]: @bensteinberg vue-select is installed, oddly

Comment (bensteinberg) [reply capstone/yarn.lock:9248]: I wonder if this is an artifact of an old Docker image cache -- if so, a subsequently-merged PR that busts the cache will pass. We don't have one yet.

Comment (ChefAndy) [reply capstone/yarn.lock:9248]: @bensteinberg #1771 might. I installed something with pip and bumped the version, if that helps.

**Pull request merged**